### PR TITLE
fix(ls-hack): Prevents a swallowed NPE when simulcast is not enabled.

### PR DESCRIPTION
**Pull request by gpolitis**

### CODE REVIEW

**Comment by paweldomas** [comment]
Where is the NPE swallowed ?

**Comment by gpolitis** [comment]
@paweldomas In AbstractRTPTranslator:227. We should print an error there.

File: src/main/java/org/jitsi/videobridge/LipSyncHack.java [hunk 1]
```diff
@@ -210,17 +211,37 @@ public void onRTPTranslatorWillWriteAudio(
...
         }
 
+        SimulcastReceiver recv = sourceVC.getTransformEngine()
+            .getSimulcastEngine().getSimulcastReceiver();
+
+        Long receiveVideoSSRC;
+        if (recv != null && recv.isSimulcastSignaled())
+        {
+            // FIXME this is a little ugly
+            receiveVideoSSRC = sourceVC.getTransformEngine()
```
Comment (paweldomas) [inline src/main/java/org/jitsi/videobridge/LipSyncHack.java]: you have "SimulcastReceiver recv" var here already or am I missing something ?

Comment (paweldomas) [inline src/main/java/org/jitsi/videobridge/LipSyncHack.java]: the whole logic in the spaghetti here fells like a candidate for a method in the VideoChannel, no ?

Comment (gpolitis) [inline src/main/java/org/jitsi/videobridge/LipSyncHack.java]: Thanks for the suggestion Pawel, I really appreciate it and I agree with you that this looks ugly. I plan to simplify it like this https://github.com/jitsi/jitsi-videobridge/blob/cc/src/main/java/org/jitsi/videobridge/LipSyncHack.java#L213 (please do not pay attention to the unchecked array accesses as the code is not finished yet).

**Pull request merged**